errbot/logs.py

```diff
@@ -20,11 +21,16 @@ def ispydevd():
 isatty = pydev or stream.isatty()  # force isatty if we are under pydev because it supports coloring anyway.
 console_hdlr = logging.StreamHandler(stream)
 
+parser = argparse.ArgumentParser()
+parser.add_argument('-D', '--dark-text', dest="dark_text", action='store_true',
+                    help='force the log output to be dark, for use with light backgrounds')
```

Comment:
I don't like how this introduces argument parsing into a module that isn't directly responsible for CLI interaction. I don't directly have a better suggestion, but I think we should find some other approach to pass on this information.